homeassistant/components/totalconnect/config_flow.py

```diff
@@ -0,0 +1,59 @@
+"""Config flow for the Total Connect component."""
+import logging
+
+from total_connect_client import TotalConnectClient
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class TotalConnectConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Total Connect config flow."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initiated by the user."""
+        errors = {}
+
+        if user_input is not None:
+            # Validate user input
+            username = user_input.get(CONF_USERNAME)
+            password = user_input.get(CONF_PASSWORD)
```

Comment:
These are required so should always be available

Reply:
removed .get()